common/app/cards/CardType.scala

```diff
@@ -12,6 +12,21 @@ sealed trait CardType {
     case ListItem => false
     case _ => true
   }
+
+  /** To actually find out if media is shown you would also need to check whether there is an image (or video) and
+    * whether image hide setting is on.
+    *
+    * But some card sizes can never show media, regardless of those options, and this is what this represents.
+    */
+  def canShowMedia = this match {
```

Comment:
can these be simple `val` or `lazy val`? Does not feel like they need to evaluate each time they are called.